SimpleKeychain/A0SimpleKeychain.h

```diff
@@ -22,6 +22,10 @@
 
 #import <Foundation/Foundation.h>
 
+#if (defined __IPHONE_8_0 || defined __MAC_10_12) && !defined __TVOS_9_0 && !defined __WATCHOS_1_0
```

Comment:
A simpler check is to see if the header is available 

```objc
#if __has_include("LocalAuthentication/LocalAuthentication.h")
```

Reply:
Great tip! I didn't know about that macro.